app/assets/javascripts/filters.js

```diff
@@ -0,0 +1,148 @@
+$(() => {
+  $('.js-filter-select').each(async (_, el) => {
+    const $select = $(el);
+    const $form = $select.closest('form');
+    const $formFilters = $form.find('.form--filter');
+    const $saveButton = $form.find('.filter-save');
+    const $isDefaultCheckbox = $form.find('.filter-is-default');
+    const categoryId = $isDefaultCheckbox.val();
+    var defaultFilter = await QPixel.defaultFilter(categoryId);
```

Comment:
consider (es+): doesn't need to be a `var` just because it's from `await` (if that was the reason) - `const` if it's a set-once value, `let` if it will be changed later.

Reply:
Yes, this should be `let`; thank's for catching that